main.js

```diff
@@ -22,6 +22,9 @@ const logSymbols = require('log-symbols')
 const { paramCase } = require('param-case')
 const { pascalCase } = require('pascal-case')
 
+const init = require('@react-native-community/cli/build/commands/init/init')
+  .default
```

Comment:
or import from one level up:

```suggestion
const init = require('@react-native-community/cli/build/commands/init').default
  .func
```